docs/messaging-service/sms-config.md

```diff
@@ -0,0 +1,28 @@
+# SMS gateway configuration
+
+Messaging service supports 2 ways of sending SMS messages out of the box:
```

Comment:
out of the box, by integrating with:
- The [CM.com SMS gateway](https://www.cm.com/sms/)
- A custom API

Reply:
fixed